server/app/views/admin/programs/ProgramBlockView.java

```diff
@@ -42,4 +51,48 @@ protected final DivTag renderProgramInfo(ProgramDefinition programDefinition) {
             editDetailsButton)
         .withClasses("bg-gray-100", "text-gray-800", "shadow-md", "p-8", "pt-4", "-mx-2");
   }
+
+  /** Renders a div presenting the predicate definition for the admin. */
+  protected final DivTag renderExistingPredicate(
```

Comment:
This is a fine incremental approach to the display we already have, but we can easily present this as a table (like the edit UI) to be much more user friendly.  Suggest a bug at least and to touch base with Miriam on what this should be, I'm unsure if it's in the mocks.

Reply:
Current implementation follows a mock from Miriam. Miriam's latest mocks abandon the tabular view, which suggests to me a tabular view here probably isn't appropriate.

> Suggest a bug at least

For what?